### PR TITLE
Add oschmod

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
@conda-forge/staged-recipes please review

File: recipes/oschmod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pywin32-on-windows  # [win]
```
Comment (ocefpaf) [inline recipes/oschmod/meta.yaml:25]: This package exist so packages that depend on `pywin32` on Windows can be noarch.

```suggestion
    - pywin32-on-windows
```

File: recipes/oschmod/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oschmod-{{ version }}.tar.gz
+  sha256: bec99216f31615ee653b2a5c87cacfb4e4b6184c0e9f71da186300dacae974f3
+
+build:
+  entry_points:
```
Comment (ocefpaf) [inline recipes/oschmod/meta.yaml:13]:
```suggestion
  noarch: python
  entry_points:
```

File: recipes/oschmod/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - oschmod = oschmod.cli:main
+    - ochmod = oschmod.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/oschmod/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/oschmod/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/oschmod/meta.yaml:24]:
```suggestion
    - python >=3.6
```

**Pull request merged**